src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward: function(distance = 1) {
```

Comment:
or this way, looks better
```suggestion
    goForward(distance = 1) {
```